### PR TITLE
Add support to Tags, Consumes, Produces and default attribute

### PR DESCRIPTION
**Pull request by alvaropaco**
### Features
Add new features:
- Tags support: tag array to the Swagger object.
- Consumes support: consumes support like Swagger ocumentation
- Produces support: produces support like Swagger ocumentation
- Default value attribute to the parameters

### CODE REVIEW

**Comment by vbarzana** [comment]
Hi guys, when can this be merged?

File: lib/generators.js [hunk 1]
```diff
@@ -59,75 +61,6 @@ function _generateDefinitions(models) {
 function _generateParameters(config, context) {
     var params = {};
 
-    //default parameters-> where, limit, skip, sort, populate, select, page
```
Comment (vbarzana) [inline lib/generators.js]: Hey mate, why did you remove the default parameters from here?

**Comment by theoomoregbee** [comment]
should be fixed in #58
